plugins/rapid7_insightappsec/setup.py

```diff
@@ -3,7 +3,7 @@
 
 
 setup(name="rapid7_insightappsec-rapid7-plugin",
-      version="1.0.3",
+      version="2.0.0",
```

Comment:
This does not need to be a major version bump. It would be if we were changing the plugin spec YAML object name (line 372 and not line 373), but the `title:` key is only cosmetic, so this is just a patch.